<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>STFTAnalysePanelClass</class>
 <widget class="QWidget" name="STFTAnalysePanelClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1301</width>
    <height>894</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>STFTAnalysePanel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox_HotCht">
       <property name="title">
        <string>整体结果</string>
       </property>
       <layout class="QVBoxLayout" name="VLayout_STFT_Cht_HotCht"/>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_HotChtPanel">
       <property name="maximumSize">
        <size>
         <width>260</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>热力图控制</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_STFTFileName">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>文件名称</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_DataStartTime">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>数据开始时间</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_dataEndTime">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>数据结束时间</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_FFTFreqInterval">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>FFT频率间隔</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_STFTGapIntervalMsec">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>STFT步长：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_FFTFreqCount">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>频率点数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_TimeFrameCount">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>时间点数</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_hotChtStartTime">
            <property name="text">
             <string>开始时间</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTimeEdit" name="timeEdit_hotChtStartTime">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="displayFormat">
             <string>mm:ss</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_hotChtTimeLength">
            <property name="text">
             <string>窗口长度</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="SBox_hotChtTimeLength">
            <property name="maximum">
             <number>20000</number>
            </property>
            <property name="value">
             <number>10000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_hotCht_startFreq">
            <property name="text">
             <string>开始频率</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="SBox_HotChtStartFreq">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maximum">
             <number>22100</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_hotCht_endFreq">
            <property name="text">
             <string>结束频率</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="SBox_HotchtEndFreq">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maximum">
             <number>22100</number>
            </property>
            <property name="value">
             <number>4500</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>颜色范围（db）</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_STFT_HotCht_StartDB">
             <property name="text">
              <string>最小幅值</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="SBox_STFT_HotCht_StartDB">
             <property name="minimum">
              <number>-20</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_STFT_HotCht_endDB">
             <property name="text">
              <string>最大幅值</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="SBox_STFT_HotCht_EndDB">
             <property name="value">
              <number>80</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_HotChtFreqBandWidth">
            <property name="text">
             <string>频率带宽（hz）</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="SBox_HotChtFreqBandWidth">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>11</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_HotChtTimeIntervalMsec">
            <property name="text">
             <string>时间带宽（ms）</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="SBox_HotChtTimeIntervalMsec">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>20</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_HotChtGenState">
          <property name="text">
           <string>刷新状态:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <item>
           <widget class="QProgressBar" name="progressBar_STFT_HotCht_Genstate">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btn_updateHotCht">
          <property name="text">
           <string>刷新</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>300</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>时间截面</string>
       </property>
       <layout class="QVBoxLayout" name="VLayout_STFT_Cht_TimeSection"/>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="minimumSize">
        <size>
         <width>260</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>260</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>时间截面图控制</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_HAxisRange">
          <property name="title">
           <string>横坐标范围（hz）</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_STFT_TimeSec_startFreq">
             <property name="text">
              <string>开始频率</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="SBox_STFT_TimeSec_startFreq">
             <property name="maximum">
              <number>22100</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_STFT_TimeSec_endFreq">
             <property name="text">
              <string>结束频率</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="SBox_STFT_TimeSec_endFreq">
             <property name="maximum">
              <number>22100</number>
             </property>
             <property name="value">
              <number>4500</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>纵坐标范围（db）</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_STFT_TimeSec_StartDB">
             <property name="text">
              <string>最小幅值</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="SBox_STFT_TimeSec_StartDB">
             <property name="minimum">
              <number>-20</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_STFT_TimeSec_endDB">
             <property name="text">
              <string>最大幅值</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="SBox_STFT_TimeSec_EndDB">
             <property name="value">
              <number>80</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="label_STFT_TimeSec_BarBandWidth">
            <property name="text">
             <string>频率格带宽（hz）</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="SBox_STFT_TimeSec_BarBandWidth">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>22100</number>
            </property>
            <property name="value">
             <number>11</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_STFTUpdateTimer">
            <property name="text">
             <string>帧刷新间隔(ms)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="SBox_STFTUpdateTimer">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>200</number>
            </property>
            <property name="value">
             <number>20</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_TimeSecGenState">
          <property name="text">
           <string>刷新状态:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QProgressBar" name="progressBar_STFT_TimeSec_Genstate">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <widget class="QCheckBox" name="checkBox_genPhaseSec">
            <property name="text">
             <string>生成相位图</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_showPhaseTimeSec">
            <property name="text">
             <string>显示相位图</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btn_updateTimeSecCht">
          <property name="text">
           <string>刷新</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>时间控制</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_STFTTimeFrameTime">
              <property name="text">
               <string>截面时间：00：00.000</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="HSlider_STFTTimeFrameIndex">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btn_lastFrame">
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>&lt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_nextFrame">
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_play">
              <property name="text">
               <string>播放</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_stop">
              <property name="text">
               <string>停止</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_jump">
              <property name="text">
               <string>跳转</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>260</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>260</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>时间界面帧信息</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_STFT_Frame_startTime">
          <property name="text">
           <string>截面开始时间</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_STFT_Frame_midTime">
          <property name="text">
           <string>截面中心时间</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_STFT_Frame_endTime">
          <property name="text">
           <string>截面结束时间</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
